bindings/flow/fdb_flow.actor.cpp

```diff
@@ -98,6 +98,81 @@ void fdb_flow_test() {
 }
 
 namespace FDB {
+	class DatabaseImpl : public Database, NonCopyable {
+	public:
+		virtual ~DatabaseImpl() { fdb_database_destroy(db); }
```

Comment:
`DatabaseImpl` is a subclass of `Database`, who is a subclass of `ReferenceCounted<Database>`. Thus that means as long as `Database` has a virtual destructor, all its subclasses down the chain does not need a virtual destructor.

Reply:
But I guess the extra virtual keyword does not really hurt anything cause it's already virtual dispatch anyways.